### PR TITLE
Added Cpu and Memory limit for master jenkins server

### PR DESCRIPTION
**Pull request by oguzy**
CPU and Mem request values are updated. With low values,
Jenkins is not able to up and running

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by oguzy** [comment]
Signed the CLA.

File: stable/jenkins/values.yaml [hunk 1]
```diff
@@ -18,8 +18,10 @@ Master:
   UseSecurity: true
   AdminUser: admin
   # AdminPassword: <defaults to random>
-  Cpu: "200m"
-  Memory: "256Mi"
+  Cpu: "2000m"
+  Memory: "2048Mi"
+  CpuLimit: "2000m"
+  MemoryLimit: "2048Mi"
```
Comment (unguiculus) [inline stable/jenkins/values.yaml:24]: Our best practice is to not specify defaults for resources at all. It should be possible to install and test charts on clusters with little resources, such as Minikube.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by viglesiasce** [comment]
Agree that we will not set the limit by default. Can you change this to add the limit but not set it in the Values file?

**Pull request closed without merging**